vm/systemSmartContracts/delegationManager.go

```diff
@@ -294,6 +300,8 @@ func (d *delegationManager) changeMinDeposit(args *vmcommon.ContractCallInput) v
 }
 
 func (d *delegationManager) getAllContractAddresses(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	//TODO: This method wasn't supposed to be called by anyone? (like in the document description)
```

Comment:
yes - it should be called by scQuery only.

Reply:
Ok, so the check should remain here?